src/smt/smt_engine.cpp

```diff
@@ -556,9 +536,11 @@ class SmtEnginePrivate : public NodeManagerListener {
 
   /** Instance of the ITE remover */
   RemoveTermFormulas d_iteRemover;
+  void finishInit();
```

Comment:
Maybe add a short comment and a space between this and `d_iteRemover` (otherwise it looks like the two belong together).